lib/foreignNodeId.test.ts

```diff
@@ -212,6 +212,7 @@ ${validationDirectionEnumTypeDefs(capitalizedName)}
               typeId2: ID! @foreignNodeId(typename: "${idsMap[2].typeName}")
               typeId3: ID! @foreignNodeId(typename: "${idsMap[3].typeName}")
               typeId4: String! @foreignNodeId(typename: "${idsMap[4].typeName}")
+              typeId5: String @foreignNodeId(typename: "${idsMap[4].typeName}")
```

Comment:
Não seria "${idsMap[5].typeName}")?

Reply:
Nope, because idsMap[5] does not exists, Actually the typeName in here does not matter, since the id will be NULL
so no conversion will be attempt.